### PR TITLE
Adding new logos and changing links

### PR DESCRIPTION
**Pull request by jsignell**
@jlstevens we can work off this branch

Screen Shot 2019-06-26 at 2 16 12 PM

### CODE REVIEW

File: doc/FAQ.rst [hunk 1]
```diff
@@ -2,51 +2,51 @@
...
 
-For the quickest response from those who can fix things, try to identify which PyViz package is most directly involved, and then click on the appropriate logo on the `PyViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``pyviz`` package itself, then please open an issue on `github.com/pyviz. <https://github.com/pyviz/pyviz/issues>`_ or chat with us on the `Gitter channel for HoloViews. <http://gitter.im/pyviz/pyviz>`_
+For the quickest response from those who can fix things, try to identify which HoloViz package is most directly involved, and then click on the appropriate logo on the `HoloViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``holoviz`` package itself, then please open an issue on `github.com/holoviz. <https://github.com/holoviz/holoviz/issues>`_ or chat with us on the `Gitter channel for HoloViews. <http://gitter.im/holoviz/holoviz>`_
```
Comment (jsignell) [inline doc/FAQ.rst]: Careful with these links.
```suggestion
For the quickest response from those who can fix things, try to identify which HoloViz package is most directly involved, and then click on the appropriate logo on the `HoloViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``holoviz`` package itself, then please open an issue on `github.com/holoviz. <https://github.com/pyviz/holoviz/issues>`_ or chat with us on the `Gitter channel for HoloViews. <http://gitter.im/pyviz/pyviz>`_
```

File: doc/FAQ.rst [hunk 2]
```diff
@@ -2,51 +2,51 @@
...
 - A set of package authors who work together to make their packages interoperate well, to help reduce gaps and duplication between packages and improve the overall user experience.
 - A set of packages that have been verified to work well together, in matching versions, that can all be installed with one command and are continuously tested to ensure that they keep working together
 - A set of examples and tutorials that help you see how various Python packages can work together to solve a huge range of visualization problems, from common situations to tricky special cases.
 
-There is a specific installable package named ``pyviz``, but it's valuable mainly for its dependencies, which are installed in matching versions when ``pyviz`` is installed.  There is almost no code actually in PyViz, apart from the examples in the tutorials.
+There is a specific installable package named ``holoviz``, but it's valuable mainly for its dependencies, which are installed in matching versions when ``holoviz`` is installed.  There is almost no code actually in HoloViz, apart from the examples in the tutorials.
 
 
-Is PyViz a commercial product?
+Is HoloViz a commercial product?
```
Comment (jbednar) [inline doc/FAQ.rst:23]:
```suggestion
How does HoloViz relate to PyViz?
=========================

The packages in HoloViz were originally described as "PyViz" packages, but in June 2019 PyViz.org was  split off as a separate general-purpose site for all of Python visualization, with HoloViz referring to the specific packages  maintained in by the group now called HoloViz.

Is HoloViz a commercial product?
```

File: doc/FAQ.rst [hunk 3]
```diff
@@ -2,51 +2,51 @@
...
 
-For the quickest response from those who can fix things, try to identify which PyViz package is most directly involved, and then click on the appropriate logo on the `PyViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``pyviz`` package itself, then please open an issue on `github.com/pyviz. <https://github.com/pyviz/pyviz/issues>`_ or chat with us on the `Gitter channel for HoloViews. <http://gitter.im/pyviz/pyviz>`_
+For the quickest response from those who can fix things, try to identify which HoloViz package is most directly involved, and then click on the appropriate logo on the `HoloViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``holoviz`` package itself, then please open an issue on `github.com/holoviz. <https://github.com/pyviz/holoviz/issues>`_ or chat with us on the `Gitter channel for HoloViews. <http://gitter.im/pyviz/pyviz>`_
```
Comment (jbednar) [inline doc/FAQ.rst]:
```suggestion
For the quickest response from those who can fix things, try to identify which HoloViz package is most directly involved, and then click on the appropriate logo on the `HoloViz home page <index.html>`_ to find their Github site for filing issues.  If you can't figure out which project is involved, or if your issue is with this website, the notebooks you downloaded from it, or the ``holoviz`` package itself, then please open an issue on `github.com/holoviz. <https://github.com/pyviz/holoviz/issues>`_ or chat with us on the `PyViz Gitter channel. <http://gitter.im/pyviz/pyviz>`_
```

**Review by jbednar — APPROVED** [review]
Including just the bokeh and mpl logos on the front page looks very strange to me; I'd favor either having only the logos of the full holoviz projects,  or else having all the previous logos, but making all of those much smaller than the ones that are fully part of holoviz.

**Comment by jsignell** [comment]
> Including just the bokeh and mpl logos on the front page looks very strange to me

Ok I'm removing them

**Comment by jsignell** [comment]
New landing page looks like:
Screen Shot 2019-06-27 at 10 46 05 AM

**Comment by jsignell** [comment]
rendered at https://pyviz-dev.github.io/holoviz/

**Pull request merged**